### DISCUSSION TITLE
Adding Scroll events

### DISCUSSION THREAD

**Discussion by TobiasHT5** [flet-dev/flet#1354] [Ideas]
I'm working on a page that has a large header section and a body. I wanted to achieve an effect where the header shrinks to a minimal version as the user scrolls down the body. 

I've tried looking up for how to do this with the current scrolling capabilities of pages and scroll enabled layouts but I can't seem to pull it off. Is there a way of doing page updates depending on scrolls? Or is it possible to include scrolling events to enable such effects?

**Comment by FeodorFitsner** [comment]
Scroll position detection/control is coming in the release this week.

Reading scroll position will be implemented as part of this: https://github.com/flet-dev/flet/issues/747 - subscribe to that issue.
